src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
```

Comment:
Why method getRandomColor returns String instead of color?

Reply:
previously i don't know how i will print Color =/ now i fix it, thx